concent_api/concent_api/settings/testing.py

```diff
@@ -65,3 +65,8 @@
 MINIMUM_UPLOAD_RATE       = 1
 
 DOWNLOAD_LEADIN_TIME      = 3
+
+# disable HandleServerErrorMiddleware in tests
+index = MIDDLEWARE.index('concent_api.middleware.HandleServerErrorMiddleware')
+if index is not None:
+    MIDDLEWARE.remove('concent_api.middleware.HandleServerErrorMiddleware')
```

Comment:
Please avoid defining variables at module level in settings (even test settings) because it's polluting the namespace. This would be better as:

``` python
if MIDDLEWARE.index('concent_api.middleware.HandleServerErrorMiddleware') is not None:
    MIDDLEWARE.remove('concent_api.middleware.HandleServerErrorMiddleware')
```

And if you absolutely have to, please give the variable a much more specific name.

Reply:
done